crates/tx-pause/src/benchmarking.rs

```diff
@@ -19,43 +19,45 @@
 
 use super::{Pallet as TxPause, *};
 
-use frame_benchmarking::benchmarks;
-
-benchmarks! {
-    pause {
-        let full_name: FullNameOf<T> = (name::<T>(b"SomePalletName"), Some(name::<T>(b"SomePalletName")));
-        // let pallet_name: PalletNameOf<T> = name::<T>(b"SomePalletName");
-        // let maybe_call_name: Option<CallNameOf<T>> = Some(name::<T>(b"some_call_name"));
-        let origin = T::PauseOrigin::try_successful_origin().unwrap();
-        // let call = Call::<T>::pause { full_name: full_name.clone() };
-        // let call = Call::<T>::pause { pallet_name: pallet_name.clone(), maybe_call_name: maybe_call_name.clone() };
-
-    }: _<T::RuntimeOrigin>(origin, full_name.clone())
-    verify {
-        assert!(TxPause::<T>::paused_calls(full_name.clone()).is_some())
-    }
+use frame_benchmarking::v2::{account, benchmarks, impl_benchmark_test_suite};
+
+#[benchmarks]
+pub mod benchmarks {
+    use frame_system::RawOrigin;
+
+    use super::*;
 
-  unpause {
-        let full_name: FullNameOf<T> = (name::<T>(b"SomePalletName"), Some(name::<T>(b"SomePalletName")));
-        let pause_origin = T::PauseOrigin::try_successful_origin().unwrap();
+    #[benchmark]
+    fn pause() {
+        let full_name = name::<T>();
+        let nominator: T::AccountId = account("recipient", 0, 2);
```

Comment:
Is this used anywhere? Also btw I think where we do need to reference accounts we should use the `whitelisted_caller()` function.